ducktape/cluster/mixed_os_cluster.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2015 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .cluster import Cluster
+from .remoteaccount import RemoteAccount
+
+
+class MixedOsCluster(Cluster):
```

Comment:
Hmm, why have this in addition to the `Cluster` class? Couldn't we just assume all clusters are potentially mixed clusters?